mpqp/tools/circuit.py

```diff
@@ -85,7 +84,7 @@ def random_circuit(
     rng = np.random.default_rng(seed)
 
     if nb_gates is None:
-        nb_gates = rng.integers(5, 10)
+        nb_gates = int(rng.integers(5, 10))
```

Comment:
better use 
```Python
if TYPE_CHECKING:
    assert isinstance(nb_gates,int)
```

Reply:
```suggestion
        nb_gates = rng.integers(5, 10)
        if TYPE_CHECKING:
            assert isinstance(nb_gates,int)
```

Reply:
I already done that before cast int, but really don't work! that's why

Reply:
I tested with the updated you done @hJaffaliColibritd, still not passing
![image](https://github.com/user-attachments/assets/a4334dff-deed-4004-80d4-3b33132fc08a)

Reply:
You right, we put it back to int and that's it

Reply:
Oh, it seems that it returns a numpy.int65